AngouriMath/Functions/Evaluation/Evaluation.Discrete.Classes.cs

```diff
@@ -335,5 +335,30 @@ protected override Entity InnerSimplify()
                 }
             }
         }
+
+        partial record Phif
+        {               
+            /// <inheritdoc/>
+            protected override Entity InnerEval()
+            {
+                if (Argument.Evaled is not Integer integer)
+                    return this;
+                else
+                {
+                    return integer.Phi();
+                }
```

Comment:
This should be a little bit refactored:
Move this code to one private method which takes an `Entity`. And inside you write `if (argument is not Integer`...
So that there is no code duplication.
And then 
```cs
... InnerEval() => InnerCompute(Argument.Evaled);
... InnerSimplify() => InnerCompute(Argument.InnerSimplified);
```

Reply:
Got it.